contracts/oracles/sources/oracle.move

```diff
@@ -0,0 +1,62 @@
+/// Oracles module is responsible for fetching various token prices via multiple oracle services
+/// 
+///  # Related Modules
+/// 
+/// There is one main operation in this module:
+/// 
+/// Creates two functions for take sui token price from supra oracle and pyth oracle.
```

Comment:
This comment is different from what I've suggested, please take a look at the suggestion.